pkg/util/node/node.go

```diff
@@ -212,3 +226,13 @@ func preparePatchBytesforNodeStatus(nodeName types.NodeName, oldNode *v1.Node, n
 	}
 	return patchBytes, nil
 }
+
+// IsNodeReady returns true if a node is ready; false otherwise.
+func IsNodeReady(node *v1.Node) bool {
```

Comment:
It seems this method is only used in this e2e test so maybe just move the method as an internal function in this file?

Reply:
Good catch, thanks.
Moved it as an internal function. PTAL